src/ga4gh/vrs/extras/translator.py

```diff
@@ -171,32 +115,12 @@ def translate_to(self, vo, fmt):
     ############################################################################
     # INTERNAL
 
-    def _get_refget_accession(self, alias):
-        """Get refget accession given alias
-
-        :param alias: Alias to get accession for
-        :return: Refget Accession if found
-        """
-        refget_accession = None
-        try:
-            aliases = self.data_proxy.translate_sequence_identifier(
-                alias, namespace="ga4gh"
-            )
-        except KeyError:
-            _logger.error("KeyError when getting refget accession: %s", alias)
-        else:
-            if aliases:
-                refget_accession = aliases[0].split("ga4gh:")[-1]
-
-        return refget_accession
-
     def _get_hgvs_tools(self):
         """ Only create UTA db connection if needed. There will be one connection per
         translator.
         """
         if self.hgvs_tools is None:
-            uta_conn = hgvs.dataproviders.uta.connect()
-            self.hgvs_tools = HgvsTools(uta_conn)
+            self.hgvs_tools = HgvsTools(self.data_proxy)
         return self.hgvs_tools
```

Comment:
IMO I think we should use the `lazy_property` decorator 

```python
from ga4gh.vrs.extras.decorators import lazy_property

@lazy_property
def hgvs_tools(self):
    """instantiates and returns an HgvsTools instance"""
    return HgvsTools(self.data_proxy)
```

Reply:
+1